### DISCUSSION TITLE
Config and logging details for Cases

### DISCUSSION THREAD

**Discussion by cyb3rz3us** [Security-Onion-Solutions/securityonion#7652] [Unsupported Versions] [answered]
One of the great things about the Security Onion documentation is that most sections have details about where to go to modify the various configs, salt pillars, and also how to access the relevant logs.  However, with Cases, I am unable to find this same information so I wanted to ask if this information exists and if so, can it be shared with the community?

As an aside, I have an issue that I believe is related to Cases but I wanted to dig into it a little first before opening a case; hence my inquiry here.

**Comment by dougburks** [comment, marked as answer]
From the Cases documentation at https://docs.securityonion.net/en/2.3/cases.html:
> Cases is a part of [Security Onion Console (SOC)](https://docs.securityonion.net/en/2.3/soc.html#soc).

If you follow that SOC link, you'll then see:
> If you’d like to customize SOC, please see the [SOC Customization](https://docs.securityonion.net/en/2.3/soc-customization.html#soc-customization) section. If you’d like to learn more about SOC logs, please see the [SOC Logs](https://docs.securityonion.net/en/2.3/soc-logs.html#soc-logs) section.

**Reply by cyb3rz3us** [reply]
Thanks @dougburks - this is my bad. In searching the docs, I just went with the first item but then didn't circle back to item 2 which was re: SOC customization.  Thanks again...
